src/tasks/runstart.ts

```diff
@@ -200,6 +200,17 @@ export const RunStartQuest: Quest = {
       do: () => visitUrl("shop.php?whichshop=september"),
       limit: { tries: 1 },
     },
+    {
+      name: "Grab Photobooth Props",
+      completed: () =>
+        get("instant_savePhotoboothProps", false) || get("_photoBoothEquipment", 0) >= 3,
+      do: (): void => {
+        cliExecute("photobooth item fake arrow-through-the-head");
+        cliExecute("photobooth item oversized monocle on a stick");
+        cliExecute("photobooth item fake huge beard");
+      },
+      limit: { tries: 1 },
```

Comment:
There can probably be a little more optimization done here:
- We can run with the assumption that the clan VIP lounge has access to the photobooth (as stated in the README), but the `oversized monocle` may not be unlocked in all clans (and may also result in this grabbing fewer than 3 items)
- The `astronaut helmet` may actually be much better than the `fake huge beard` if the user has a censer and isn't saving embers

It might be better with something like
```
cliExecute("photobooth item fake arrow-through-the-head"); // First guaranteed item
cliExecute("photobooth item oversized monocle on a stick");
if (!get("instant_saveEmbers", false) && have($item`Sept-Ember Censer`))
    cliExecute("photobooth item astronaut helmet");
cliExecute("photobooth item fake huge beard"); // Second guranteed item (if we have not hit 3 items yet)
cliExecute("photobooth item photo booth supply list"); // Third guranteed item (if we have not hit 3 items yet)
```

But I'd think 
(1) Generating a list of items to grab by priority (and necessity),
(2) Filtering that list down to 3 items, and
(3) Looping through that list to acquire said items
is probably much cleaner

Reply:
Ultimately, you'd want to do that - but you'd need an accounting of what someone has, whether stats for stat tests matter, whether NC matters, whether three free kills takes them over a threshold, etc. There's also no command for checking if things are unlocked (yet?), Mafia assumes your clan has everything and doesn't tell you otherwise.

I think future state such an optimizer would be good! It's not possible yet.